transport_nantes/topicblog/forms.py

```diff
@@ -5,7 +5,45 @@
 from mailing_list.models import MailingList
 
 
-class TopicBlogItemForm(ModelForm):
+class ModelFormTemplateList(ModelForm):
+    """Render a ModelForm and pre-select template_name field's initial value.
+
+    If a value was already selected (e.g. in case of edit), the value is left
+    alone and preselected.
+    In case of a new object, the 1st template is pre-selected.
+
+    This class implies an implementation of template_config and that the
+    instance has a template_name attribute.
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.set_initial_template()
+
+    def get_template_list(self) -> list:
+
+        templates = self.instance.template_config
+        template_list = \
+            [(None, "Selectionnez un template ...")]
```

Comment:
The first element in the list is now being selected, which is good.
But this is a non choice: what could it possibly mean for someone to choose it?

Reply:
When I do pull down the dropdown, I see "Selectionnez un template ..." in the dropdown.  It's not selected, but why is it even there?  It's never the right thing to select.